virtio-devices/src/transport/pci_device.rs

```diff
@@ -287,13 +287,14 @@ struct VirtioPciDeviceState {
 
 impl VersionMapped for VirtioPciDeviceState {}
 
+type QueueTuples = Vec<(usize, Queue<GuestMemoryAtomic<GuestMemoryMmap>>, EventFd)>;
```

Comment:
I believe this is clippy being overzealous here. I would just place an override on this line. This does not enhance readability.

Reply:
Haha yes it is clippy bothering me :)
Ok I'll override the clippy warning.